docs/src/"troubleshooting.md"

```diff
@@ -0,0 +1,24 @@
+# Troubleshooting
+This page will contain errors that have been seen before, and how they were fixed. The goal is that solutions to common problems are found here. 
+By "local REPL" a REPL running on the same computer as the host is meant. By "remote REPL", a REPL running on a different computer is meant.
+
+### IOError: connect: connection refused (ECONNREFUSED)
+This error has been encountered when
+1) Running `connect_repl()`, while attempting to connect to a local REPL. The problem was that no local REPL had previously run `serve_repl()`. To fix this, run `serve_repl()` in the local REPL.
+2) Running `connect_remote()`, while attempting to connect to a local REPL. The problem was that no local REPL had previously run `serve_repl()`. To fix this, run `serve_repl()` in the local REPL.
```

Comment:
Let's merge these as they're the same.

```suggestion
1) Running `connect_repl()` or `connect_remote()`, while attempting to connect to a local REPL. The problem was that no local REPL had previously run `serve_repl()`. To fix this, run `serve_repl()` in the local REPL.
```

Reply:
I felt like one of the strengths of my proposal was that any combination of issue and error was covered. Merging looses that benefit, right?